frontend/app/src/components/history/filtering/FilterDropdown.vue

```diff
@@ -80,21 +79,32 @@ watch([keyword, suggestion], async ([keyword, suggestion]) => {
   }
 
   const search = splitSearch(keyword);
+  const exclude =
+    'string' in suggestion && suggestion.allowExclusion
+      ? search.exclude || false
+      : false;
```

Comment:
There should be a way to simplify this than having two false fallbacks `else` clauses